gordon/plugins_loader.py

```diff
@@ -202,6 +210,10 @@ def load_plugins(config, plugin_kwargs):
         config.
     """
     installed_plugins = _gather_installed_plugins()
+    metrics_plugin = _get_metrics_plugin(config, installed_plugins)
+    if metrics_plugin:
+        plugin_kwargs['metrics'] = metrics_plugin
+
```

Comment:
Is there a need for an `if metrics_plugin` check? or rather, will this check be removed once the metric logger is implemented?

Could you also return the metrics plugin? The router will need it. It could just be a part of the list of inited plugins that are returned, and I can just pull it out by checking if it provides the interface.

Reply:
I'd like Gordon to be able to run with no metric plugin configured.  Unless we don't want to do that, in which case we should explicitly raise an error somewhere.

Reply:
@ocsw - @matt0bi's PR (#23) is the default metrics provider, so it should always run with a metrics provider. @descent3me and I also chatted a bit and agreed that plugins can assume there will always be a metrics provider, so there's no `if self.metrics` littered throughout. The instrumentation I'm doing right now requires that core have a metrics provider too (since we want to provide some default metrics out of the box and not rely on plugin providers to do _everything).

Reply:
ok

Reply:
I referenced this comment in #23. I'd be happy to update this part in as part of #23 since I think it'll be easier to address and test when there are multiple relays to pick from.